### DISCUSSION TITLE
(Tutorial) How to find the right recovery for your system

### DISCUSSION THREAD

**Discussion by kerta1n** [sebanc/brunch#982] [Show and tell]
ChromeOS is one of the best lightweight Linux distributions that was closed-source. But thanks to the hard work of @sebanc , you all can install ChromeOS on almost any Intel computer you have. However, you should remember that because we are tricking Google into thinking that we have a legit, manufactured Chromebook, for which, AUP expiry dates still apply. So, I'll show you how to find the best recovery image so that you can still receive updates for your install.

First off, you want to find the right recovery for your CPU. Get your CPU model, and then head over to Amazon and search "chromebook intel (your model here)". If you're lucky enough, you may have a Chromebook with your exact same CPU. However, because it is old hardware, Google has most likely put that Chromebook model on the "bye-bye updates" list. 

Next, in another tab, go to [this](https://support.google.com/chrome/a/answer/6220366) link, and find your model that you found on Amazon. If it is within 2 years of expiring, it's not worth downloading. Then, head back over to Amazon, and look for another one. Check if it is within 2 years, and if it is not, then you can continue to the next step. If it is within 2 years, repeat the process, and remember that it doesn't have to be the exact same CPU model number, but rather just the same CPU series (I'll explain this more in the example.) 

The third step is to now go to [cros.tech](https://cros.tech), and download your recovery. You can now download brunch and continue with your install. Happy flashing!

Example: I have a Windows Dell Inspiron 3542 laptop which has a Celeron N3050. The Chromebooks I had with the same CPU were the HP Chromebook 11 G5 Enterprise Edition and the Lenovo N22 Chromebook. However, both of them expire June 2022, which is next year. Instead of using Setzer, I found Octopus, which still supports Celeron processors but incldes the N4000 and N4100 CPU series. I haven't switched to Octopus, but I will update this when I do.

**Comment by drom1988** [comment]
I feel like I'm losing my mind, I've tried many different recovery files but no matter what the installation goes fine but I get stuck on the chrome os logo upon booting. I'm trying to install on an Intel NUC DCCP847DYE, Processor is an Intel Celeron 847. I've tried Octopus, Coral, Samus and Caroline. Any suggestions?

**Reply by kerta1n** [reply]
It's been a long time since I've messed with brunch, but try letting it sit for at least 3h.

**Reply by drom1988** [reply]
Thanks, I'll give that a try over the weekend.

**Reply by maxed4901** [reply]
Inorder to easily find out out which chomeos you need, you have to do some research. If ur device came with eg; *intel i5 4th gen processor*, go to google and search for "chromebooks with i5 4th gen" processors. If your device supports touch screen,  and specifying screen size helps narrowing down your failure to boot attempts. 

Goto cros.tec and search for your device. 

I have a surfacepro 3 which is dualbooted with chromeos 120 by using the ekko image. It works awesome..

**Comment by gjlamb** [comment]
Is this tutorial still valid?  I'm trying to get ChromeOS on my Surface Pro 6.  A lot of other doco says to use Shyvana recovery but if I follow this and search Amazon for an intel i5 8250U all I get is a HP 15 Chromebook but it doesn't match up with anything in CrOS.tech.

I've installed brunch on a couple of other devices and it worked well.  Having a major issue this time on the SP6.  Going down in recovery and Brunch releases to see if I can connect to Wifi.  Seen in other discussions that 126 has a problem with password protected wifi networks on Flex so assume the same problem will be here in Brunch?

**Reply by maxed4901** [reply]
Here are some Chromebooks that match your criteria of having an i5 8th gen. 

1. **HP Chromebook x360 12b**:
   - **Processor**: Intel Core i5-8250U (8th Gen)
   - **GPU**: Intel UHD Graphics 620
   - **Screen Size**: 12 inches
   - **Touch Screen**: Yes
   - **Year of Manufacture**: 2019

2. **Acer Chromebook Spin 11 CP311**:
   - **Processor**: Intel Core i5-8250U (8th Gen)
   - **GPU**: Intel UHD Graphics 620
   - **Screen Size**: 11.6 inches
   - **Touch Screen**: Yes
   - **Year of Manufacture**: 2018

These models have the touch screen feature and match your other requirements.

Check the Recoveries for these models. I've decided to move on with fydeos (there's a special version for surface pros) + in the long term there aren't updates and if u update with brunch enforce, there's a good chance it will not reboot.

**Reply by gjlamb** [reply]
Thanks for the info. I realized I was also doing something wrong that wasn't documented in any of the instructions. Still had issues with not being able to log into my mesh network but I could log in to my main provider router. 

Also had issues with Play store service constantly crashing. Decided to put Win11 back on it and found a pre-owned Pixelbook nearby to try.  I've always wanted one so snapped ot up. 

I looked at FydeOS as it seemed to do almost everything I wanted but one thing which was a must have for me was Phone Hub.

**Comment by guilhermetb** [comment]
Hi,
Which bin board from the website https://cros.tech/ should I use for the notebook:
Dell Inspiron N4050
Core i3 2310M 2100Mhz Sandy Bridge
4gb DDR3-1333 DDR3
Intel HD Grafics 3000
SSD 120GB
Drive CDROM
Bios version A08
Serial number 3LJDSR1

I've already tried the versions:
samus 83 89 91 - No video
asuka - The video appears but it is buggy,
rammus - It gets frozen on the GoogleOS logo

In asuka and rammus ctrl + alt + f2 enter in localhost login:, i type root and have prompt brunch~#

I am using MBR support
